Arduino Yun/Main/Main.ino

```diff
@@ -0,0 +1,55 @@
+#include <Bridge.h>
+#include <BridgeServer.h>
+#include <BridgeClient.h>
+#include <Servo.h>
+
+BridgeServer server;
+
+void setup() {
+  pinMode(13, OUTPUT);
+  
+  Serial.begin(9600);
+  Bridge.begin();
+
+  server.listenOnLocalhost();
+  server.begin();
+}
+
+void loop() {
+  BridgeClient client = server.accept();
+  if(client){
+    process(client);
+    client.stop();
+  }
+  delay(50);
+}
+
+void process(BridgeClient client){
+  String command = client.readStringUntil('/');
+  command = command.substring(0,command.length()-2);
```

Comment:
Ok. That's fine for now then. Please research the multiple files option though / possibilities of classes. Once your code grows it might become a bigger problem. I suggest you make an issue for it so you won't forget ;)